### DISCUSSION TITLE
New Version

### DISCUSSION THREAD

**Discussion by arunjayaraj1988** [WinMerge/winmerge#552] [Q&A] [answered]
My requested change was implemented by Winmerge team and gave me a per user setup.  But I am currently looking for per-machine setup. Could you please let me know when the new version of Winmerge will be released? Any tentative date?

**Comment by sdottaka** [comment, marked as answer]
I'd like to release this month, but I haven't set a specific date.

Please use version 2.16.9 beta installer or CI build artifact until it is released.

https://sourceforge.net/projects/winmerge/files/beta/2.16.9/WinMerge-2.16.9-x64-Setup.exe/download
https://ci.appveyor.com/project/sdottaka/winmerge/build/artifacts

**Reply by JadranR** [reply]
Thank you :-)))

Best regards,
Jadran
